hibernate-core/src/main/java/org/hibernate/collection/spi/BagSemantics.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or http://www.gnu.org/licenses/lgpl-2.1.html
+ */
+package org.hibernate.collection.spi;
+
+/**
+ * @author Andrea Boriero
+ */
+public interface BagSemantics<B> extends CollectionSemantics<B> {
+}
+
```

Comment:
seems the content from another PR is included

Reply:
Good catch @NathanQingyangXu, going to fix it. Thanks